spec/lib/tasks/resanitize.rake_spec.rb

```diff
@@ -0,0 +1,120 @@
+require "spec_helper"
+
+describe "rake resanitize:all" do
+  include ActiveJob::TestHelper
+
+  shared_examples "resanitizes" do |field|
+    field_sanitizer_version = "#{field}_sanitizer_version"
+
+    context "when the #{field} attribute has an old sanitizer version" do
+      it "removes invalid content and updates the sanitizer version" do
+        record.update_columns(field => "<invalid>",
+                              field_sanitizer_version => ArchiveConfig.SANITIZER_VERSION - 1)
+
+        expect do
+          perform_enqueued_jobs { subject.invoke }
+          record.reload
+        end.to output.to_stdout
```

Comment:
`output.to_stdout` is not checking for any string specifically, can we remove it?

Reply:
I added it because it suppresses the rake task's printouts when running tests, but if you're okay with more verbose tests I can drop it.

Reply:
Nice use of that. It's fine to stay - there's a lot more stdout prints than most tested rake tasks right?

Reply:
It's not too big (~5-10 lines per test), but because it's being run 60+ times, it adds up.